src/lib/computeDiskPercent.ts

```diff
@@ -0,0 +1,11 @@
+import memoize from 'lodash/memoize'
+
+const compute = (usage: number | string, quota: number | string): string => {
+  const parsedUsage = parseInt(`${usage}`, 10)
+  const parsedQuota = parseInt(`${quota}`, 10) || 100000000000
+  const percent = (+parsedUsage / +parsedQuota) * 100
+
+  return `${percent % 1 ? percent.toFixed(2) : percent}`
+}
+
+export const computeDiskPercent = memoize(compute)
```

Comment:
Why we don't re-use what we already have? https://github.com/cozy/cozy-settings/blob/master/src/components/StorageView.jsx#L21-L27 

Or at least having only one way to deal with that? Here we have two pieces of code, this is not good.

Reply:
Yes my aim was to extract the function from the view, I will refactor the old view so it uses the new decoupled function

Reply:
Yes please 👍

Reply:
Updated, but storage view still does some calculations of its own that are unrelated to percentage